src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contacts in the address book.
+ */
+public class Contact {
+    
+    public String value;
+    protected boolean isPrivate;
+    
+    public Contact (boolean isPrivate) {
+        this.isPrivate = isPrivate;
```

Comment:
initialize value.
Consider passing the validation regex and the validation message.